lib/src/package.dart

```diff
@@ -139,66 +139,64 @@ class Package {
     return processResult.exitCode;
   }
 
-  /**
-   * Moves the packages folder into the root of the web folder. WARNING: this may
-   * change in the future versions dartdoc-viewer.
-   *
-   */
-  void moveDocumentationPackages(Version version) {
-    String out = join(BUILD_DOCUMENTATION_ROOT_PATH, "${name}-${version}",
-        DARTDOC_VIEWER_OUT);
-    String webPath = join(out, 'web');
-    String webPackagesPath = join(webPath, 'packages');
-    String outPackagesPath = join(out, 'packages');
-
-    // 1) remove symlink in out/web/packages
-    Directory webPackagesDirectory = new Directory(webPackagesPath);
-    webPackagesDirectory.deleteSync();
-
-    // 2) only copy dartdoc_viewer specific packages
-    _moveDartDocViewerSpecificFiles(outPackagesPath, webPackagesPath);
-  }
-
-  void _moveDartDocViewerSpecificFiles(String outPackagesPath, String webPackagesPath) {
-    // mkdir web/packages
-    // copy -r packages/web_components web/packages/
-    // copy -r packages/polymer web/packages/
-
-    Directory webPackagesDirectory = new Directory(webPackagesPath);
-    webPackagesDirectory.createSync();
-
-    String outWebComponentsPath = join(outPackagesPath, "web_components");
-    String outPolymerPath = join(outPackagesPath, "polymer");
-    String outDartdocViewerPath = join(outPackagesPath, "dartdoc_viewer");
-
-    String webWebComponentsPath = join(webPackagesPath, "web_components");
-    String webPolymerPath = join(webPackagesPath, "polymer");
-    String webDartdocViewerPath = join(webPackagesPath, "dartdoc_viewer");
-
-    Directory outWebComponentsDirectory = new Directory(outWebComponentsPath);
-    Directory outPolymerDirectory = new Directory(outPolymerPath);
-    Directory outDartdocViewerDirectory = new Directory(outDartdocViewerPath);
-
-    outWebComponentsDirectory.renameSync(webWebComponentsPath);
-    outPolymerDirectory.renameSync(webPolymerPath);
-    outDartdocViewerDirectory.renameSync(webDartdocViewerPath);
-  }
+  // /**
+  //  * Moves the packages folder into the root of the web folder.
+  //  *
+  //  * WARNING: this may change in the future versions dartdoc-viewer.
```

Comment:
Yeah, the output from dartdoc isn't a dart app w/ `packages/` folders, but just raw html+css. No more separate docgen and dartdoc-viewer.